src/Handler/SentryHandler.php

```diff
@@ -12,12 +12,12 @@
 use Monolog\Handler\AbstractProcessingHandler;
 use Monolog\Logger;
 use Sentry\Severity;
-use Sentry\State\Scope;
 use SilverStripe\Core\Config\Configurable;
 use SilverStripe\Core\Injector\Injectable;
 use PhpTek\Sentry\Log\SentryLogger;
 use PhpTek\Sentry\Adaptor\SentryAdaptor;
 use PhpTek\Sentry\Adaptor\SentrySeverity;
+use Throwable;
```

Comment:
I'm trying to stick to a certain import order (granted, it isn't being adhered-to here)

1. <PHP>
2. <SilverStripe>
3. <3rdParty>
4. <Module>

If you feel like it therefore, `Throwable` should be imported first :-)

Reply:
My IDE just alphabetises them lol. :) I can arrange.

Reply:
Fixed.